src/Adapter/Currency/CommandHandler/AddCurrencyHandler.php

```diff
@@ -72,6 +80,15 @@ public function handle(AddCurrencyCommand $command)
             $entity->iso_code = $command->getIsoCode()->getValue();
             $entity->active = $command->isEnabled();
             $entity->conversion_rate = $command->getExchangeRate()->getValue();
+            // CLDR locale give us the CLDR reference specification
+            $cldrLocale = $this->localeRepoCLDR->getLocale($this->defaultLanguage->locale);
```

Comment:
As far as I remember, the locale was sometimes empty.

What do you think about

```suggestion
            $cldrLocale = $this->localeRepoCLDR->getLocale($this->defaultLanguage->getLocale());
```